test/functional/rpc_psbt.py

```diff
@@ -160,11 +158,11 @@ def run_test(self):
         node1_addr = self.nodes[1].getnewaddress()
         node2_addr = self.nodes[2].getnewaddress()
         txid1 = self.nodes[0].sendtoaddress(node1_addr, 13)
-        txid2 =self.nodes[0].sendtoaddress(node2_addr, 13)
-        self.nodes[0].generate(6)
+        txid2 = self.nodes[0].sendtoaddress(node2_addr, 13)
+        blockhash = self.nodes[0].generate(6)[0]
         self.sync_all()
-        vout1 = find_output(self.nodes[1], txid1, 13)
-        vout2 = find_output(self.nodes[2], txid2, 13)
+        vout1 = find_output(self.nodes[1], txid1, 13, blockhash = blockhash)
+        vout2 = find_output(self.nodes[2], txid2, 13, blockhash = blockhash)
```

Comment:
Same here.

Reply:
how do you catch these? are you running a linter on just the changed code?

Reply:
You are free to use a formatter on the diff, see https://github.com/bitcoin/bitcoin/blob/master/doc/productivity.md#writing-code. However, we do not enforce a specific style, because developers use different editors and commit workflows.

@practicalswift I think that purely stylistic feedback about code is not useful (especially when the pull request has already been reviewed and would thus require a re-review). Though, it is fine to leave stylistic feedback as part of a feedback that has actual concerns about the code.

Reply:
ah, missed that, thanks. 

hm, I hear your point about stylistic feedback & reviews, but I think the feedback should be directed to me instead of practicalswift. If other reviewers made comments that caused me to make deeper code changes, I could incorporate the whitespace updates. But not worth re-committing for just that. ✔️Noted. 

In this case changes to master required an update to `rpc_psbt` anyways.